work12/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="./style.css" type="text/css" />
+  <title>TerraceTechフロントエンドエンジニア養成所</title>
+</head>
+
+<body>
+  <div class="wrap" id="js-wrap">
+    <ul class="list" id="js-list"></ul>
+  </div>
+
+  <div id="js-button-wrap" class="button-wrap"></div>
```

Comment:
Instead of placing an empty element here in html, I think you can insert it at the position specified by `js` (for example, before or after `js-wrap`).
By doing so, I think it will be better without extra elements.

Reply:
Creating a button with js made the html cleaner.
thank you.

commit: [cac07ad](https://github.com/yasu-pro/js-handson/pull/12/commits/cac07ad5097ac60ae01e46f6024dcf08cfc08d5b)